docs/commands.md

```
@@ -0,0 +1,235 @@
+
+```bash
+# GitLab project
+ztigit environments -P "devops/deploy-tools"
+
+# GitHub repo
+ztigit environments -P "zsoftly/ztiaws" -p github
+```
+
+Output:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)